crates/aptos-crypto/src/bls12381/bls12381_sigs.rs

```diff
@@ -113,6 +113,18 @@ impl Signature {
 
         self.verify_aggregate_arbitrary_msg(&msgs_refs, pks)
     }
+
+    /// Return a dummy signature.
```

Comment:
Can you briefly mention here where/how this is used?

Reply:
some tests just  require A signature